client/src/containers/backend/Dashboard.js

```diff
@@ -88,7 +96,11 @@ class DashboardContainer extends PureComponent {
                       {'Activity'} <i className="manicon manicon-pulse-small"></i>
                     </h3>
                   </header>
-                  <DashboardComponents.Activity />
+                  { this.props.statistics ?
```

Comment:
This check should be in the DashboardComponents.Activity component, not outside of it. By placing it outside of the component, the check has to happen every time it's used. Inside of it, it only happens once. In the render method of DashboardComponents.Activity, just return null if there's no this.props.statistics value.